pgx_async/bin/pgx_async_example.ml

```diff
@@ -9,7 +9,7 @@ module Employee = struct
         id SERIAL PRIMARY KEY,
         name VARCHAR(100) NOT NULL UNIQUE);
     |}
-    |> Deferred.ignore
+    |> Deferred.ignore_m
```

Comment:
What changed with `ignore`?

Reply:
It was just renamed to `ignore_m`

From their docs:
```
val ignore_m : 'a t -> unit t
ignore_m t is map t ~f:(fun _ -> ()). ignore_m used to be called ignore, but we decided that was a bad name, because it shadowed the widely used Caml.ignore. Some monads still do let ignore = ignore_m for historical reasons.
```